spec/requests/content_item_requests/message_bus_spec.rb

```diff
@@ -141,4 +141,73 @@
       end
     end
   end
+
+  context "/v2/publish" do
+    let(:request_body) { { update_type: "major" }.to_json }
+    let(:request_path) { "/v2/content/#{content_id}/publish" }
+    let(:request_method) { :post }
+
+    before do
+      draft = FactoryGirl.create(:draft_content_item, v2_content_item)
+      FactoryGirl.create(:version, target: draft, number: 1)
+    end
+
+    it "sends a message with the 'format.update_type' routing key" do
+      Timecop.freeze do
+        do_request
+
+        expect(response.status).to eq(200)
+
+        expected_payload = v2_content_item.except(:access_limited).merge(
+          update_type: "major",
+          transmitted_at: DateTime.now.to_s(:nanoseconds),
+        ).to_json
+
+        delivery_info, _, payload = wait_for_message_on(@queue)
+
+        expect(delivery_info.routing_key).to eq("guide.major")
+        expect(JSON.parse(payload)).to eq(JSON.parse(expected_payload))
+      end
+    end
+  end
+
+  describe "content type inlining" do
```

Comment:
Quick correction - rummager doesn't do any HTML tag stripping - but I think some publisher apps which are sending content to rummager do before they send it.

Regardless, I believe having only the HTML is fine for indexing - but we definitely need to know if a field is expected to contain HTML markup, so we can strip it out (or possibly make use of it to index some bits of text as being more important - eg, headings).

I think the current design is that we'd be able to find that out by reading the schema, but not simply by inspecting the document itself without reference to the schema. This is probably fine, though does make it a bit hard to manage if the schema changes from a field being a plain text field to being a govspeak + html field - there's no way for our indexing code to know about the change, and thus to ensure that it's stripping out HTML tags for all HTML fields.  This is probably something we'll just have to live with, though.